doc/guidelines/frontend-keyhandling.md

```diff
@@ -0,0 +1,90 @@
+# Frontend Keybindings
+
+This document gives guidelines on how to implement keybindings in the frontend.
+
+### Keybinding Configuration
+
+The file 'keybindings.json' contains all keybindings that are used in the application and serves the following characteristics:
+
+- keybindings can easily be changed by editing the file
+- developers have an overview of all keybindings
+- there are different sections for keybindings:
+  - global: keybindings that are globally active
+  - Scopes for specific layers:
+    Keybindings that are only active if the corresponding layer is active.
+    These scopes should be named according to the layer name, postfixed with `_layer`.
+  - _just_for_documentation_:
+    Keybindings that are only used for documentation purposes since the keybinding is hardcoded in the component.
+    For example, the search field can be cleared by pressing the Escape key, which is typically a native browser action and should not be configurable in the keybindings file.
+- modifier keys are supported (Shift, Alt, Ctrl, Meta)
+  - they can be used in combination with other keys by using the '+' sign (e.g. 'Alt+P', 'Shift+A')
```

Comment:
How do these map on Mac OS?
Can you maybe introduce a wildcard modifier key so that it automatically works on Mac OS, too?

Reply:
the react key event supports the four mentioned modifier keys (Shift, Alt, Ctrl, Meta). 
The keys should be mapped like this:

- event.ctrlKey: Represents whether the Control key (Ctrl) is pressed.
- event.shiftKey: Represents whether the Shift key is pressed.
- event.altKey: Represents whether the Alt key is pressed (Option key on macOS).
- event.metaKey: Represents whether the Meta key (Command key on macOS) is pressed.

Unfortunately, i cannot test if it is really working like this. I created a left over issue: https://github.com/ElektraInitiative/PermaplanT/issues/1057